test/functional/sqlsrv/sqlsrv_encrypted_query_nosqltype.phpt

```diff
@@ -4,6 +4,22 @@ Test using sqlsrv_query for binding parameters with column encryption and a cust
 <?php require('skipif_not_ksp.inc'); ?>
```

Comment:
is this test ksp specific?

Reply:
No it's not. I originally wrote it for testing ksp, but no longer required. Thanks!